docs/source/enterprise-guide/change-management.mdx

```diff
@@ -0,0 +1,468 @@
+---
+title: Change Management in Apollo Federation
+sidebar_title: Change Management
+---
+
+A federated GraphQL architecture is a multi-tiered architecture made up of
+independent components. GraphQL requests flow through multiple components to
+construct a response, which in turn flows back through those same components, as
+illustrated below:
+
+```mermaid
+graph LR
+    X(Client) --> A(Gateway)
+    A -.-> X
+    A --> B(Subgraph A)
+    B -.-> A
+    A --> C(Subgraph B)
+    C -.-> A
+    B --> D(REST API)
+    D -.-> B
+    D --> E(Database)
+    E -.-> D
+    C --> F(gRPC API)
+    F -.-> C
+    F --> G(Third party API)
+    G -.-> F
+```
+
+The ability to update components independently is a key benefit of federation.
+However, understanding and de-risking changes to a component can be challenging,
+especially as changes may cascade and require changes in other components. This
+is especially true for GraphQL schema defintions, which must be coordinated
+between subgraph services and the gateway.
+
+This section covers the seven most common change management scenarios in a
+federated architecture. We'll consider the proper sequencing of release events,
+how Apollo tooling makes managing changes easier, and when progressive delivery
+techniques like blue-green deploys are useful.
+
+Specifically, we'll examine scenarios that:
+
+**Involve a schema change**
```

Comment:
Might make this a bulleted list with _sub_-lists:

* **Involve a schema change**
    * Backward compatible etc.